results/Snowflake__snowflake-arctic-embed-m-v2.0/f2a7d59d80dfda5b1d14f096f3ce88bb6bf9ebdc/model_meta.json

```diff
@@ -0,0 +1 @@
+{"name": "Snowflake/snowflake-arctic-embed-m-v2.0", "revision": "f2a7d59d80dfda5b1d14f096f3ce88bb6bf9ebdc", "release_date": null, "languages": null, "n_parameters": null, "memory_usage": null, "max_tokens": null, "embed_dim": null, "license": "apache-2.0", "open_weights": null, "public_training_data": false, "public_training_code": null, "framework": ["PyTorch"], "reference": null, "similarity_fn_name": null, "use_instructions": null, "training_datasets": null, "adapted_from": null, "superseded_by": null, "loader": null}
```

Comment:
a lot of the metadata seems to be missing here - is it not run using the implementation in the package? (loader: null)

Reply:
It is using the package but without https://github.com/embeddings-benchmark/mteb/pull/1574. It still used the `sentence_transformers_loader`. I let these models ran while I was making the other PR and then once the PR was finished I just ran the models again on a single task to validate that the previous results stayed the same. I'll upload the new metadata.